values.yaml

```diff
@@ -531,6 +531,8 @@ kube-prometheus-stack:
     image:
       repository: mtr.devops.telekom.de/kubeprometheusstack/grafana
       tag: 10.1.4
+    initChownData:
```

Comment:
Just wondering that we have it before. Maybe the default value is changed, which didn't work in secure environment, of course, https://github.com/grafana/helm-charts/blob/main/charts/grafana/values.yaml#L355-L386